Wrappers/Python/cil/plugins/ccpi_regularisation/functions/regularisers.py

```diff
@@ -80,17 +80,25 @@ def check_input(self, input):
         pass
 
 class TV_Base(RegulariserFunction):
+
```

Comment:
The docstring is missing. Is this because it is a "base" class? However, the input parameter should be documented.

Reply:
Updated to new style doc format.